catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java

```diff
@@ -327,15 +330,22 @@ public boolean onContextItemSelected(MenuItem item) {
 
 	@Override
 	public void onSpriteChecked() {
+		Log.d("Lausi", "ONSpriteChecked");
```

Comment:
> Btw: "Lausi" is not a good tagname  , more precisely this line should be removed

And now you know why I commented on this line in the first place. :stuck_out_tongue_winking_eye: